components/case-studies-card/case-studies-card.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Image from 'next/image';
+
+import style from './case-studies-card.module.scss';
+
+type Props = {
+  linkUrl?: string;
+  linkText?: string;
+  logo: string;
+  title: string;
+  text: string;
+};
+
+export default function CaseStudiesCard({
+  title,
+  text,
+  logo,
+  linkUrl,
+  linkText,
+}: Props) {
+  console.log('logo: ', logo);
```

Comment:
I think this was from me. Sorry!

Reply:
yep no worries, I've deleted this in my latest commit.